Snippets/Autofac/Extensions.DependencyInjection_1/Usage.cs

```diff
@@ -0,0 +1,17 @@
+﻿class Usage
+{
+    Usage()
+    {
+        #region Autofac
```

Comment:
@DavidBoike Is it enough for them to be here or do I actually have to show them on https://github.com/Particular/docs.particular.net/blob/master/nservicebus/dependency-injection/autofac_usage_extensions.dependencyinjection_%5B1%2C).partial.md?

Reply:
If you have a snippet, you have to use it.

if the Extensions.DependencyInjection "major version" doesn't use a specific snippet, then delete the snippet.

From the `//not used` here it looks like this entire file should be deleted. I'm guessing the only snippet you'll need is "this is how to use Autofac using Extensions.DependencyInjection"